app/controllers/api/teams.php

```diff
@@ -1103,14 +1114,13 @@
         $dbForProject->purgeCachedDocument('users', $user->getId());
 
         if ($membership->getAttribute('confirm')) { // Count only confirmed members
-            Authorization::skip(fn () => $dbForProject->decreaseDocumentAttribute('teams', $team->getId(), 'total', 1, 0));
+            Authorization::skip(fn () => $dbForProject->increaseDocumentAttribute('teams', $team->getId(), 'total', 1));
```

Comment:
shouldn't this be decrease? Best to restore it to what it was before.

Reply:
@stnguyen90 I'm a bit confused about that, it says if `$membership->getAttribute('confirm')`, doesn't that mean we _should_ count that member?

Reply:
I'll commit it to reverse, but I was confused by that

Reply:
Because it was a confirmed member, the total was incremented previously. So, now in the delete membership, the total should be decremented.